src/checkfiles/checkfiles.py

```diff
@@ -361,6 +360,31 @@ def worker(job):
     return file_validation(*job)
 
 
+def patching_worker(job):
+    portal_uri = job[0]
+    portal_auth = job[1]
+    file_validation_record = job[2]
+    submitted_md5sum = job[3]
```

Comment:
Several variables are not used, for example: submitted_md5sum, output_type, file_format_type, assembly

Reply:
thank you!